PubnativeLite/PubnativeLite/VAST/PNLiteVASTPlayerViewController.m

```diff
@@ -761,4 +762,15 @@ - (void)contentInfoViewWidthNeedsUpdate:(NSNumber *)width {
     [self.view layoutIfNeeded];
 }
 
+#pragma mark - Utils: check for bundle resource existance.
```

Comment:
We could have created a Util class for it but since it is only in 2 places & the tremendous effort that we've all put in since yesterday, i think we're okay 😂

Reply:
yes, it was a quick util function.